docs/pages/access-controls/guides/headless.mdx

```diff
@@ -102,6 +103,14 @@ $ tctl create -f cap.yaml
 
 ## Step 2/3. Initiate Headless WebAuthn
 
+First open a terminal on the remote machine and confirm the `tsh` binary 
```

Comment:
Is "remote machine" the machine from which the user is trying to authenticate or the upstream server?

Reply:
This is a bit confusing, because headless authentication basically splits the process across multiple machines.

The _headless_ machine that wants to perform some action initiates the action and waits for authn, and another machine completes the authentication flow so that the original machine can be unblocked.

Reply:
You would need to have a terminal open yourself or a command run from that remote machine.  This is confirming that you have `tsh` installed and it's the right version.  Prior to this update we don't call out that you are doing this on the remote machine.  I do think a diagram would help here which I will try to put in later.